packages/webapi/controllers/corecontracts/controller.go

```diff
@@ -66,34 +58,13 @@ func (c *Controller) addAccountContractRoutes(api echoswagger.ApiGroup, mocker i
 		SetOperationId("accountsGetAccountNonce").
 		SetSummary("Get the current nonce of an account")
 
-	api.GET("chain/core/accounts/nftdata/:nftID", c.getNFTData).
-		AddParamPath("", params.ParamNFTID, params.DescriptionNFTID).
-		AddParamQuery("", params.ParamBlockIndexOrTrieRoot, params.DescriptionBlockIndexOrTrieRoot, false).
-		AddResponse(http.StatusUnauthorized, "Unauthorized (Wrong permissions, missing token)", authentication.ValidationError{}, nil).
-		// AddResponse(http.StatusOK, "The NFT data", mocker.Get(isc.NFTJSON{}), nil).
-		SetOperationId("accountsGetNFTData").
-		SetSummary("Get the NFT data by an ID")
-
-	api.GET("chain/core/accounts/token_registry", c.getNativeTokenIDRegistry).
-		AddParamQuery("", params.ParamBlockIndexOrTrieRoot, params.DescriptionBlockIndexOrTrieRoot, false).
-		AddResponse(http.StatusUnauthorized, "Unauthorized (Wrong permissions, missing token)", authentication.ValidationError{}, nil).
-		AddResponse(http.StatusOK, "A list of all registries", mocker.Get(models.NativeTokenIDRegistryResponse{}), nil).
-		SetOperationId("accountsGetNativeTokenIDRegistry").
-		SetSummary("Get a list of all registries")
-
-	//nolint:unused
-	type foundryOutputParams struct {
-		chainID      string `swagger:"required,desc(ChainID (Hex Address))"`
-		serialNumber uint32 `swagger:"required,min(1),desc(Serial Number (uint32))"`
-	}
-
-	api.GET("chain/core/accounts/foundry_output/:serialNumber", c.getFoundryOutput).
-		AddParamPathNested(foundryOutputParams{}).
+	api.GET("chain/core/accounts/objectdata/:objectID", c.getObjectData).
+		AddParamPath("", params.ParamObjectID, params.DescriptionObjectID).
 		AddParamQuery("", params.ParamBlockIndexOrTrieRoot, params.DescriptionBlockIndexOrTrieRoot, false).
 		AddResponse(http.StatusUnauthorized, "Unauthorized (Wrong permissions, missing token)", authentication.ValidationError{}, nil).
-		AddResponse(http.StatusOK, "The foundry output", mocker.Get(models.FoundryOutputResponse{}), nil).
-		SetOperationId("accountsGetFoundryOutput").
-		SetSummary("Get the foundry output")
+		AddResponse(http.StatusOK, "The object data", mocker.Get(isc.IotaObject{}), nil).
```

Comment:
Not sure what is the proper type to return here. Should we return the content in BCS directly?